packages/system-tests/assets/lwc-recipes/force-app/main/default/lwc/contactList/contactList.html

```diff
@@ -0,0 +1,21 @@
+<template>
```

Comment:
should we have our own version of test files instead of sharing with system-tests? just worried about depending on another package and potential noise later

Reply:
I agree.  We definitely should change the way the integration tests pick up sample workspaces.  This change should be included with the multi-root refactor.